### PR TITLE
[PM-3804] Remove Server Flag Icons

### PR DESCRIPTION
**Pull request by rr-bw**
## Type of change

- Tech debt (refactoring, code cleanup, dependency upgrades, etc)

## Objective

To remove flag icons from the environment selector

## Code changes

- Remove flag icons
    - `apps/web/src/app/components/environment-selector/environment-selector.component.html`
    - `libs/angular/src/auth/components/environment-selector.component.html`

- Update styles for the Browser and Desktop dropdown
    - `apps/browser/src/popup/scss/environment.scss`
    - `apps/desktop/src/scss/environment.scss`

- Remove `hasFlags` input binding
    - `apps/web/src/app/components/environment-selector/environment-selector.component.ts`
    - `apps/web/src/app/auth/trial-initiation/trial-initiation.component.html`
    - `apps/web/src/app/layouts/frontend-layout.component.html`

- Change self-hosted translation to lowercase to align with `bitwarden.com` and `bitwarden.eu`
    - `apps/browser/src/_locales/en/messages.json`
    - `apps/desktop/src/locales/en/messages.json`

- Remove translations that are no longer necessary
    - `apps/web/src/locales/en/messages.json`

### Web - Login
Screenshot 2023-09-06 at 12 33 11 PM

### Web - Trial Initiation

Screenshot 2023-09-06 at 12 33 43 PM

### Browser

Screenshot 2023-09-06 at 1 42 34 PM

### Desktop

Screenshot 2023-09-06 at 1 42 18 PM

### CODE REVIEW

**Review by JaredSnider-Bitwarden — APPROVED** [review]
LGTM!

**Review by ike-kottlowski — APPROVED** [review]
Good work.

File: apps/browser/src/popup/scss/environment.scss [hunk 1]
```diff
@@ -68,7 +68,7 @@ html.browser_safari {
     background-color: themed("boxBackgroundColor");
   }
   padding: 5px;
-  width: 100%;
+  width: 200px;
```
Comment (djsmith85) [inline apps/browser/src/popup/scss/environment.scss:71]: Why do we need to set a specific width on browser and desktop. Shouldn't a flexible width be better than a fixed?

Comment (rr-bw) [reply apps/browser/src/popup/scss/environment.scss]: My mistake, I was testing something there. Reverted back to 100%.

**Review by djsmith85 — APPROVED** [review]
Changes are looking good.

**Review by ike-kottlowski — APPROVED** [review]

**Pull request merged**